pkg/agent/svid/rotator.go

```diff
@@ -84,7 +85,11 @@ func (r *rotator) shouldRotate() bool {
 }
 
 // rotateSVID asks SPIRE's server for a new agent's SVID.
-func (r *rotator) rotateSVID(ctx context.Context) error {
+func (r *rotator) rotateSVID(ctx context.Context) (err error) {
+	counter := telemetry.StartCall(r.c.Metrics, &err, "svid", "rotate")
```

Comment:
nit: perhaps this should be `agent_svid` to distinguish it from workload svid rotation without having to refer to a label value